src/test/java/io/airlift/slice/BenchmarkSlice.java

```diff
@@ -44,11 +48,18 @@ public Object compareTo(BenchmarkData data)
         return data.slice1.compareTo(0, data.slice1.length(), data.slice2, 0, data.slice2.length());
     }
 
+    @Benchmark
+    public Object equalsUnchecked(BenchmarkData data)
+            throws Throwable
+    {
+        return data.slice1.equalsUnchecked(0, data.slice2, 0, data.slice1.length());
+    }
+
     @State(Scope.Thread)
     public static class BenchmarkData
     {
         @Param({"1", "7", "8", "16", "32", "64", "127", "32779"})
-        private int size;
+        private int size = 1;
```

Comment:
Inital call of test method in `main` (line 95) fails because size is 0 in this case.